### PR TITLE
[4.0] Group contact toolbar buttons

### PR DESCRIPTION
**Pull request by bembelimen**
### Summary of Changes
This PR continues the grouping of action buttons in the backend (see https://github.com/joomla/joomla-cms/pull/22832, https://github.com/joomla/joomla-cms/pull/22393, https://github.com/joomla/joomla-cms/pull/23337 and https://github.com/joomla/joomla-cms/pull/23336). This time the contact buttons.

### Testing Instructions
Go to the contact manager in the backend and check the toolbar. There should be a "Change status" dropdown now which activates when you select a contact item.

### CODE REVIEW

**Comment by Hackwar** [comment]
I have tested this item :white_check_mark: successfully on 3d2feb46c71b7cda1fe84e0b93cb627c0a958745<br /><br />Tested successfully<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23367">issues.joomla.org/tracker/joomla-cms/23367</a>.</sub>

**Comment by PhocaCz** [comment]
Tested in 4.0.0-alpha6-dev successfully :ok: :heavy_check_mark:


OK

**Comment by PhocaCz** [comment]
I have tested this item :white_check_mark: successfully on 4e39b829cd68e8e70e200e9c5901e657430f93ab<br /><br />Tested now in Joomla! Issue Tracker - CMS<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23367">issues.joomla.org/tracker/joomla-cms/23367</a>.</sub>

File: administrator/components/com_contact/View/Contacts/HtmlView.php [hunk 1]
```diff
@@ -150,54 +151,81 @@ protected function addToolbar()
...
-			ToolbarHelper::checkin('contacts.checkin');
+			$dropdown = $toolbar->dropdownButton('status')
+				->text('JTOOLBAR_CHANGE_STATUS')
+				->toggleSplit(false)
+				->icon('fa fa-globe')
+				->buttonClass('btn btn-info')
+				->listCheck(true);
+
+			$childBar = $dropdown->getChildToolbar();
+
```
Comment (Quy) [inline administrator/components/com_contact/View/Contacts/HtmlView.php:174]:
```suggestion
```

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23367">issues.joomla.org/tracker/joomla-cms/23367</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**